app/javascript/pages/MyEvents/IndividualEvents.jsx

```diff
@@ -0,0 +1,237 @@
+import React, { useState } from 'react'
+import * as R from 'ramda'
+import moment from 'moment'
+
+import ActionDone from 'material-ui/svg-icons/action/done'
+import ActionInfoOutline from 'material-ui/svg-icons/action/info-outline'
+import AVNotInterested from 'material-ui/svg-icons/av/not-interested'
+import ReactTable from 'react-table'
+
+import { Modal, Close } from '@zendeskgarden/react-modals'
+import { Button } from '@zendeskgarden/react-buttons'
+
+import CreateEditModalContents from './CreateEditModalContents'
+import DeleteModalContents from './DeleteModalContents'
+import { NoEventsMessage, EventsTable } from './StyledComponents'
+
+import styled from 'styled-components'
+
+const eventStatusIcon = event => {
+  switch (event.status) {
+    case 'REJECTED':
+      return <AVNotInterested color="#A93A38" />
```

Comment:
Should we use garden color variables here?

Reply:
I'll see if we can do that :)

Reply:
changed the icons to garden icons 😃